src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,180 @@
+package duke.task;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import duke.exception.DukeIoException;
+import duke.exception.DukeTaskNumberOutOfBoundsException;
+import duke.util.DateTimeUtils;
+import duke.util.FileUtils;
+
+/**
+ * The is the TaskList class that that
+ * contains a list of task.
+ */
+public class TaskList {
+    private static final String DIR_NAME = "data";
+    private static final String FILE_NAME = "duke.txt";
+    private final List<Task> tasks;
+
+    /**
+     * Constructs a TaskList object.
+     */
+    public TaskList() {
+        tasks = new ArrayList<>();
+    }
+
+    /**
+     * Adds task to TaskList.
+     *
+     * @param task Task.
+     */
+    public void addTask(Task task) {
+        tasks.add(task);
+    }
+
+    /**
+     * Returns size of tasks.
+     *
+     * @return Tasks size.
+     */
+    public int size() {
```

Comment:
Names representing methods should be verbs, i.e. getSize() (Under coding standard)